### ISSUE TITLE
Rule idea: Comments end with a period (.)

### ISSUE THREAD

**Issue by ram4nd** [eslint/eslint#8450]
**Please describe what the rule should do:**
The best readability of comments would be achieved when they start with capital letter and end with a period like regular sentences. There is a rule for capital first letter, but no rule for punctuation.

**What category of rule is this? (place an "X" next to just one item)**

[x] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Comment by kaicataldo** [comment]
Not sure how useful this is given we can't also determine if the contents are a complete sentence or not.

Contrived example, but what about a comment like the following?
```js
/* Here's some list:
 * - item A
 * - item B
 */
```

Would the rule then require it to be written as the following?
```js
/* Here's some list:
 * - item A
 * - item B.
 */
```

Or a real world example, would it require a period for the following comment? https://github.com/eslint/eslint/blob/master/lib/api.js#L1-L4

**Comment by platinumazure** [comment]
I'm wondering if this could/should be generalized to a sort of `comment-pattern` rule which would, to oversimplify a bit, take one or more regexes and flag all comments that don't match any of the supplied regexes?

**Comment by kaicataldo** [comment]
Are you suggesting replacing the capitalized-comments rule as well?

**Comment by kaicataldo** [comment]
Whoops, sorry, hit the wrong button!

**Comment by platinumazure** [comment]
@kaicataldo I didn't mean to suggest that, no. That could be tougher to replace since we added a few configuration options for that rule to make it easier to use. So I'm not sure what I would want to do there.

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get enough support from the team and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after a long time tend to never do it, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.
